LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe-expected.txt

```diff
@@ -1,4 +1,4 @@
 
 
-PASS navigate event for navigation.back() - same-document in an iframe
+FAIL navigate event for navigation.back() - same-document in an iframe promise_test: Unhandled rejection with value: undefined
```

Comment:
The problem with all of these AFAICT, is that in WebKit promise->reject(RejectAsHandled); actually causes a rejection.

I guess Chromium has a way to mark a promise as handled with resolve.